olp-cpp-sdk-authentication/include/olp/authentication/UserAccountInfoResult.h

```diff
@@ -0,0 +1,366 @@
+                  activity and will unlock after some time. */
+    kEnabled   /*!< The account is enable and the user is allowed to log. */
+  };
+
+  /**
+   * @brief Gets HERE Account user ID.
+   *
+   * @return HERE Account user ID.
+   */
+  const std::string& GetUserId() const { return user_id_; }
+
+  /**
+   * @brief Sets HERE Account user ID.
+   *
+   * @param user_id HERE Account user ID.
+   */
+  void SetUserId(std::string user_id) { user_id_ = std::move(user_id); }
+
+  /**
+   * @brief Gets the realm in which the user account exists.
+   *
+   * @return The user realm.
+   */
+  const std::string& GetRealm() const { return realm_; }
+
+  /**
+   * @brief Sets the realm in which the user account exists.
+   *
+   * @param realm The user realm.
+   */
+  void SetRealm(std::string realm) { realm_ = std::move(realm); }
+
+  /**
+   * @brief Gets Facebook ID of the user.
+   *
+   * @note Not empty only if the account was created as a result of signing up
+   * with a Facebook token.
+   *
+   * @return Facebook ID of the user.
+   */
+  const std::string& GetFacebookId() const { return facebook_id_; }
+
+  /**
+   * @brief Sets Facebook ID of the user.
+   *
+   * @param facebook_id Facebook ID of the user.
+   */
+  void SetFacebookId(std::string facebook_id) {
+    facebook_id_ = std::move(facebook_id);
+  }
+
+  /**
+   * @brief Gets firstname of the user.
+   *
+   * @return Firstname of the user.
+   */
+  const std::string& GetFirstname() const { return firstname_; }
+
+  /**
+   * @brief Sets firstname of the user.
+   *
+   * @param firstname Firstname of the user.
+   */
+  void SetFirstname(std::string firstname) {
+    firstname_ = std::move(firstname);
+  }
+
+  /**
+   * @brief Gets lastname of the user.
+   *
+   * @return Lastname of the user.
+   */
+  const std::string& GetLastname() const { return lastname_; }
+
+  /**
+   * @brief Sets lastname of the user.
+   *
+   * @param lastname Lastname of the user.
+   */
+  void SetLastname(std::string lastname) { lastname_ = std::move(lastname); }
+
+  /**
+   * @brief Gets primary email address.
+   *
+   * @return Primary email address.
+   */
+  const std::string& GetEmail() const { return email_; }
+
+  /**
+   * @brief Sets primary email address.
+   *
+   * @param email Primary email address.
+   */
+  void SetEmail(std::string email) { email_ = std::move(email); }
+
+  /**
+   * @brief Gets recovery email address.
+   *
+   * @return Recovery email address.
+   */
+  const std::string& GetRecoveryEmail() const { return recovery_email_; }
+
+  /**
+   * @brief Sets recovery email address.
+   *
+   * @param recovery_email Recovery email address.
+   */
+  void SetRecoveryEmail(std::string recovery_email) {
+    recovery_email_ = std::move(recovery_email);
+  }
+
+  /**
+   * @brief Gets day of birth in the Day/Month/Year format.
+   *
+   * @return Day of birth.
+   */
+  const std::string& GetDob() const { return dob_; }
```

Comment:
Why not use full name 'GetDayOfBirth' ?

Reply:
It's a common abbreviation but I can change it, no problem

Reply:
The field is called dob in the OpenAPI so please keep `GetDob` and `SetDob`.

```json
{
  ...
  "dob": "31/01/1980",
  ...
}
```